index.html

```diff
@@ -0,0 +1,302 @@
+        information. See section [[[#protocol-registry]]].
+      </dd>
+      <dt>
+        <dfn data-for="Digital identity">Issuer</dfn>
+      </dt>
+      <dd>
+        The entity that issues a [=digital identity=], such as a government
+        agency or certified organizations.
+      </dd>
+    </dl>
+    <h2>
+      Scope
+    </h2>
+    <p>
+      The following items are within the scope of this specification:
+    </p>
+    <ul>
+      <li>Requesting a [=digital identity=], including mechanisms for secure
+      presentation.
+      </li>
+    </ul>
+    <p>
+      The following items are out of scope:
+    </p>
+    <ul>
+      <li>Enrollment process for establishing a [=digital identity=].
+      </li>
+      <li>UI/UX considerations, with the exception of privacy considerations,
+      which are addressed to ensure the protection of user data during the
+      request process.
+      </li>
+    </ul>
+    <h2>
+      Extensions to the `Navigator` interface
+    </h2>
+    <pre class="idl">
+    partial interface Navigator {
+      [SecureContext, SameObject] readonly attribute CredentialsContainer identity;
+    };
+    </pre>
+    <h3>
+      The `identity` attribute
+    </h3>
+    <p data-dfn-for="Navigator">
+      The <dfn>identity</dfn> attribute provides access to the the underlying
+      {{CredentialsContainer}} for managing [=identity credentials=].
+    </p>
+    <h3>
+      Extensions to Credential Management API
+    </h3>
+    <aside class="issue" data-number="65"></aside>
+    <h2>
+      Extensions to `CredentialRequestOptions` dictionary
+    </h2>
+    <pre class="idl">
+    partial dictionary CredentialRequestOptions {
+      sequence&lt;IdentityRequestProvider&gt; providers;
+    };
+    </pre>
+    <h3>
+      The `providers` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="CredentialRequestOptions">providers</dfn> member
+      is a sequence of [=digital identity/request protocol=] that can
+      potentially be handled by a user's selected [=identity credential
+      provider=].
+    </p>
+    <h2>
+      The `IdentityRequestProvider` dictionary
+    </h2>
+    <p>
+      The {{IdentityRequestProvider}} dictionary is used to specify a [=digital
+      identity/request protocol=] and structured request, which the user agent
+      MAY match against a [=identity credential provider=].
+    </p>
+    <pre class="idl">
+    dictionary IdentityRequestProvider {
+      required DOMString protocol;
+      required ArrayBuffer request;
+    };
+    </pre>
+    <h3>
+      The `protocol` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="IdentityRequestProvider">protocol</dfn> member
+      denotes the [=digital identity/request protocol=] when requesting an
+      identify credential.
+    </p>
+    <p>
+      The {{IdentityRequestProvider/protocol}} member's value is be one of the
+      keys defined in [[[#protocol-registry]]].
```

Comment:
Let's keep protocol as a DOMString because we are going to move the Registry to another document (which will make updating the enum painful once this is on the W3C Rec Track). We can add a note that it should be implemented as an enum, and mimic enum behavior via prose. We do the same thing in the Permission spec.

Reply:
SGTM